File: java/OneLoop.java
import java.util.ArrayList;

class OneLoop
{
	public static boolean Duplicates(ArrayList<Integer> list){
		for(int i = 0; i < list.size(); i++){
			for(int j = i; j < list.size(); j++){
				if(i != j && list.get(i) == list.get(j)){
					return true;
				}
			}
		}
		return false;
	}

    public static void main(String[] args)
    {
        ArrayList<Integer> list = new ArrayList<Integer>();
	list.add(2);
	list.add(4);
	list.add(3);
	System.out.println(Duplicates(list));
    }
};

File: python/one_loop.py
def one_loop(l: list) -> bool:
    i = 0
    while i < len(l):
        j = 0
        while j < len(l):
            if j != i and l[j] == l[i]:
                return True
            j = j + 1
        i = i + 1
    return False

if __name__ == "__main__":
    print(one_loop([1,2,3,4]));
